src/tools/wasm2js.cpp

```diff
@@ -658,7 +658,7 @@ Ref AssertionEmitter::emitAssertReturnFunc(Builder& wasmBuilder,
   std::unique_ptr<Function> testFunc(
     wasmBuilder.makeFunction(testFuncName,
                              std::vector<NameType>{},
-                             body->type,
+                             Signature(Type::none, body->type),
```

Comment:
How did this work before?

Reply:
This builder API used to take only the function return type, `body->type`. Now it takes a HeapType implicitly constructed from the function signature.